image/php-fpm/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM ubuntu
+
+# ADD /laravel /var/www
```

Comment:
BTW, having an appreciation and understanding for this issue totally gives you +1 do your innate docker skills.